include/rocksdb/utilities/options_type.h

```diff
@@ -660,17 +764,26 @@ Status ParseVector(const ConfigOptions& config_options,
   result->clear();
   Status status;
 
+  // Turn off ignore_unknown_objects so we can tell if the returned
+  // object is valid or not.
+  ConfigOptions copy = config_options;
+  copy.ignore_unsupported_options = false;
```

Comment:
Should we mention this change in HISTORY.md?

Reply:
I do not think it is necessary.  The options_type is only in the public API to satisfy some of the Configurable interfaces and is not really for public usage.

Reply:
At least, we should mention the behavior changes in the comment of ParseVector.

Reply:
There is no behavior change from the outside of this method.  This allows errors reported from inside the method to be differentiated.  Previously, if ignore_unsupported_options=true, the status below would return OK at times that were not OK (because we ignored the error) -- we would not detect NotFound in the inner loop.  Now the inner loop can return NotFound and the outer code can handle it appropriately,